packages/collection/src/preview.js

```diff
@@ -157,3 +159,75 @@ export const ParametricLayoutPreview = ( props ) => {
 		</div>
 	)
 };
+
+export const CarouselLayoutPreview = ( props ) => {
+  const {
+    attributes,
+    posts,
+  } = props;
+
+  const {
+    isLandscape,
+    showMedia,
+    showMeta,
+    showTitle,
+    showDescription,
+    showButtons,
+    showPagination,
+    carouselLayout,
+    postsToShow,
+    postsToShowPerRow,
+  } = attributes;
+
+  const cardProps = {
+    placeholder: true,
+    hasFixedAspectRatio: true,
+    isLandscape,
+    showMedia,
+    showMeta,
+    showTitle,
+    showContent: showDescription,
+    showButtons,
+  };
+
+  let fixedOptions = {
+    dots: showPagination,
+    infinite: true,
+    slidesToShow: postsToShowPerRow,
+  };
+
+  let settings = {
+    dots: showPagination,
+    infinite: true,
+    variableWidth: true
+  };
+
+  if ( carouselLayout !== 'variable' ) {
+    settings = fixedOptions;
+  }
+
+  return (
+    <CollectionPreview hasAppender={ false } { ...props }>
+      <div className="block-editor-inner-blocks">
+        <div className="block-editor-block-list__layout" >
+          <Slider {...settings}>
+            {
+              !! posts && posts.map( ( post, idx ) => {
+                return (
+                  <PostCard key={ idx } post={ post } isLandscape={ isLandscape } attributes={ attributes } />
+                );
+              } )
+            }
+            {
+              ! posts && Array.from( Array( postsToShow ).keys() ).map( ( obj, idx ) => {
+                return (
+                  <Card key={ idx } { ...cardProps } />
+                )
+              } )
+            }
```

Comment:
Even though, at the moment, the `CarouselLayoutPreview` doesn't actually use the `Card` component, at some point in the future it could. But, instead of duplicating the snippet of code from `ClassicLayoutPreview` we should try to be more DRY and abstract this snippet of code into a reusable component.